pkg/liqo-controller-manager/offloadingStatus-controller/remoteconditions_management.go

```diff
@@ -0,0 +1,85 @@
+package offloadingstatuscontroller
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+
+	offv1alpha1 "github.com/liqotech/liqo/apis/offloading/v1alpha1"
+	mapsv1alpha1 "github.com/liqotech/liqo/apis/virtualKubelet/v1alpha1"
+	liqoconst "github.com/liqotech/liqo/pkg/consts"
+	liqoutils "github.com/liqotech/liqo/pkg/utils"
+)
+
+// chooseRightRemoteCondition selects the right remote condition in according to the remote namespace
+// phase obtained by means of NamespaceMap.Status.CurrentMapping.
+func chooseRightRemoteCondition(phase mapsv1alpha1.MappingPhase) offv1alpha1.RemoteNamespaceCondition {
```

Comment:
```suggestion
func mapPhaseToRemoteNamespaceCondition(phase mapsv1alpha1.MappingPhase) offv1alpha1.RemoteNamespaceCondition {
```

Just a suggestion, feel free to discard

Reply:
it sounds clearer, ty